docs/farming.md

```diff
@@ -1,12 +1,13 @@
-# Subspace Farmer
 
-This is the documentation/guideline on how to run the farmer. Also, in the very end, you can find the [glossary](#glossary-for-farm-commands) for 
-various farm commands. 
+# 👨‍🌾 Getting Started Farming
+
+>This documentation will only cover connecting to the Farmnet, though you may run an independant chain if you would like. Please refer to the [Substrate Guide](https://docs.substrate.io/) for more information on how to develop on substrate based nodes.
```

Comment:
Not sure if this is something helpful here though...

Reply:
typo in `independant`, should be `independent`